src/Files.App/Views/LayoutModes/StandardLayoutMode.cs

```diff
@@ -276,11 +272,8 @@ protected async void RenameTextBox_KeyDown(object sender, KeyRoutedEventArgs e)
 						NextRenameIndex = 0;
 						EndRename(textBox);
 
-						if
-						(
-							newIndex >= 0 &&
-							newIndex < ListViewBase.Items.Count
-						)
+						if (newIndex >= 0 &&
```

Comment:
Parenthesis should be on their own line

Reply:
Oh, why on earth should be on? This is pretty old writing style of C language and I'm quite sure shouldn't be like this.

Reply:
I was thinking we'd want to do the same as we do for brackets.

Reply:
I think waste of lines...

Reply:
👍